fpo-api/api/send_email.py

```diff
@@ -8,30 +9,34 @@
 from email.mime.text import MIMEText
 from email.header import Header
 from email.utils import formataddr
+import requests
+import base64
 
 from smtplib import SMTP, SMTPException
 
 LOGGER = logging.getLogger(__name__)
 
-
-def send_email(recipient_email: str, pdf_data: bytes, pdf_name: str):
-    server_addr = settings.SMTP_SERVER_ADDRESS
+def send_email(recipient_email: str, pdf_data: bytes, pdf_name: str, auth_token: str)->{}:
     sender_email = settings.SMTP_SENDER_EMAIL
-    sender_name = settings.SMTP_SENDER_NAME
-    if not server_addr:
-        LOGGER.debug("SMTP server address not configured")
-        return
+    #sender_name = settings.SMTP_SENDER_NAME
```

Comment:
This being blank will cause an error to be logged and the email will not be sent.

Reply:
My mistake.  I see the check has been removed.  Can the `sender_name` be combined with the `sender_email` like;  `sender_name<sender_email>`?  So we're not just getting a raw email address coming across.